### PR TITLE
feat: Hyperledger Fabric Context

### LINKED ISSUE #11: Hyperledger Fabric Context
Batch Writer will reference Hyperledger Fabric implementation of blockchain client and CAS client.

### PR DESCRIPTION
**Pull request by sandrask**
Batch Writer will reference Hyperledger Fabric implementation of blockchain client and CAS client.

Closes #11

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

File: cmd/sidetree-server/main.go [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+	"github.com/trustbloc/sidetree-core-go/pkg/processor"
+
+	"github.com/go-openapi/runtime"
+	"github.com/trustbloc/sidetree-fabric/pkg/context"
+	"github.com/trustbloc/sidetree-node/pkg/requesthandler"
+	"github.com/trustbloc/sidetree-node/restapi/operations"
+)
+
+var logger = logrus.New()
+var config = viper.New()
```
Comment (fqutishat) [inline cmd/sidetree-server/main.go]: why viper new is static

File: pkg/context/context_test.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+func TestNewSDKConfigError(t *testing.T) {
+	config := viper.New()
+
+	config.Set(keyConfigFile, "./invalid/config.yaml")
+	config.Set(keyProtocolFile, protocolConfigFile)
+
+	sctx, err := New(config)
+	assert.NotNil(t, err)
```
Comment (fqutishat) [inline pkg/context/context_test.go]: change to require

File: cmd/sidetree-server/main.go (hunk 1 above)
Comment (sandrask) [reply cmd/sidetree-server/main.go]: Fixed

File: pkg/context/context_test.go (hunk 2 above)
Comment (sandrask) [reply pkg/context/context_test.go]: Fixed

File: cmd/sidetree-server/main.go [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+	"github.com/trustbloc/sidetree-core-go/pkg/dochandler/didvalidator"
+	"github.com/trustbloc/sidetree-core-go/pkg/processor"
+
+	"github.com/go-openapi/runtime"
+	"github.com/trustbloc/sidetree-fabric/pkg/context"
+	"github.com/trustbloc/sidetree-node/pkg/requesthandler"
+	"github.com/trustbloc/sidetree-node/restapi/operations"
+)
+
+const didDocNamespace = "did:sidetree:"
```
Comment (rolsonquadras) [inline cmd/sidetree-server/main.go]: Would this did method name("did:sidetree:") be constant for all instances/deployments of sidetree solutions ?

Comment (sandrask) [reply cmd/sidetree-server/main.go]: In protocol spec they use only this method/namespace name. Reference node implementation
has it is configured in yaml file.

**Pull request merged**